_layouts/category.html

```diff
@@ -80,6 +80,10 @@
   	font-size: 0.7em;
   	color: #545454;
   }
+  .returnChevron {
```

Comment:
I adjusted based on your feedback for the returnChevron css class (now return-chevron). Thanks!